recipes/cytocad/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cytocad" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 22289cdca9ebf5627dda3acc74f869c4cd73421eed41eb6680505f6e077d6e0a
+
+build:
+  skip: True # [py<36 or osx]
```

Comment:
remove the python skip here and add this pinning to the host section please

Reply:
Ok, thanks!